frontend/src/features/homes/services/group-manager.js

```diff
@@ -0,0 +1,75 @@
+import { $axios } from "src/boot/axios";
+import { Notify } from "quasar";
+import { errorNotifier } from "src/utils/notifier.js";
+import { i18n } from "src/boot/i18n";
+
+export const getCurrentManagers = async groupId => {
+  try {
+    const response = await $axios.post("/methods/getCurrentManagersApi", {
+      groupId
+    });
+    return response.data;
+  } catch (error) {
+    errorNotifier(error);
+    return [];
+  }
+};
+
+export const getEligibleManagers = async currentManagers => {
+  try {
+    const response = await $axios.post("/methods/getEligibleManagerListApi", {
+      idsToFilter: currentManagers
+    });
+    return response.data;
+  } catch (error) {
+    errorNotifier(error);
+    return [];
+  }
+};
+
+export const assignManager = async ({ groupId, users }) => {
+  try {
+    await $axios.post("/methods/assignManager", {
+      groupId,
+      users
+    });
+    Notify.create({
+      position: "top-right",
+      type: "positive",
+      message: i18n.t("managerAssign-create-successful")
+    });
```

Comment:
Services should have a single responsibility: fetching data. Any additional logic, such as showing UI elements, should probably reside outside of the service.

Reply:
some of these APIs can be reused and the same notifiers can be used.
moved notifier to util as it is common for all.